components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1103 @@
+& + \int_A \rho_0 \left[ \tilde{w}_{tr} - \tilde{ u} \right] dA \\
+& - \int_A \rho_0 \left[ \tilde{w}_{tr} - \tilde{ u} \right] dA
+= 0
+$$ (vh-mass-pseudo)
+
+**Tracer:**
+
+$$
+\frac{d}{dt} \int_A \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}_k^{\text{top}}}  \varphi \, d\tilde{z} \, dA
+& + \int_{\partial A} \left( \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}_k^{\text{top}}}  \varphi \, {\bf u} \, d\tilde{z} \right) \cdot {\bf n}_\perp \, dl & \\
+& + \int_A \left\{ \rho_0 \varphi \left[\tilde{w}_{tr} - \tilde{ u} \right] \right\}_{\tilde{z} = \tilde{z}^{\text{top}}} \, dA & \\
+& - \int_A \left\{ \rho_0 \varphi \left[\tilde{w}_{tr} - \tilde{ u} \right] \right\}_{\tilde{z} = \tilde{z}^{\text{bot}}}\, dA
+& = 0
+$$ (vh-tracer-pseudo)
+
+**Momentum:**
+
+$$
+\frac{d}{dt} \int_{A} \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}^{\text{top}}} \, {\bf u} \, d\tilde{z} \, dA
+&+
+\int_{\partial A} \left( \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}^{\text{top}}} \, {\bf u} \otimes {\bf u} \, d\tilde{z} \right) \cdot {\bf n}_\perp \, dl \\
+&+
+\int_A \rho_0 \, {\bf u} \left[ \tilde{w}_{tr} - \tilde{ u} \right]_{\tilde{z} = \tilde{z}^{\text{top}}} \, dA
+-
+\int_A \rho_0 \, {\bf u} \left[ \tilde{w}_{tr} - \tilde{ u} \right]_{\tilde{z} = \tilde{z}^{\text{bot}}} \, dA \\
+&=
+\int_A \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}^{\text{top}}} {\bf b}_{\perp} \, d\tilde{z} \, dA
++
+\int_{\partial A} \left( \int_{\tilde{z}^{\text{bot}}}^{\tilde{z}^{\text{top}}} \frac{{\bf \tau}}{\rho} \, d\tilde{z} \right) dl \\
+&\quad
++ \int_A \left[ \frac{{\bf \tau}}{\rho} \right]_{\tilde{z} = \tilde{z}^{\text{top}}} \, dA
+- \int_A \left[ \frac{{\bf \tau}}{\rho} \right]_{\tilde{z} = \tilde{z}^{\text{bot}}} \, dA
+$$ (vh-momentum-pseudo)
+
+These equations express horizontal and vertical fluxes naturally in terms of pseudo-height, enabling fully consistent discretization in the vertical coordinate used for prognostic evolution.
+
+## 7. Vertical Discretization for the Layered Equations
+
+The previous equation set [](#vh-mass-pseudo) to [](#vh-momentum-pseudo) is for an arbitrary layer bounded by $\tilde{z}^{\text{top}}$ above and $\tilde{z}^{\text{bot}}$ below. We now provide the details of the vertical discretization. The ocean is divided vertically into $K_{max}$ layers, with $k=0$ at the top and increasing downwards (opposite from $z$). Layer $k$ is bounded between $\tilde{z}_k^{\text{top}}$ above and $\tilde{z}_{k+1}^{\text{top}}$ below (i.e. $\tilde{z}_k^{\text{bot}} = \tilde{z}_{k+1}^{\text{top}}$).
+
+The layer thickness of layer $k$, used in MPAS-Ocean, is
+
+$$
+h_k = \int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} dz.
+$$ (def-thickness)
+
+In Omega we will use the pseudo-thickness,
+
+$$
+{\tilde h}_k(x,y,t)
+&= \int_{{\tilde z}_{k+1}^{\text{top}}}^{{\tilde z}_k^{\text{top}}} d\tilde{z} \\
+&= \frac{1}{\rho_0} \int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} \rho \, dz \\
+&= \frac{1}{\rho_0 g} \left( \hat{p}_{k+1}^{\text{top}} - \hat{p}_k^{\text{top}} \right)
```

Comment:
Why are we using/introducing $\hat{p}$ here? Isn't it $p$ as used in the equations above?